subprojects/core/src/main/java/org/gradle/util/WrapUtil.java

```diff
@@ -44,7 +44,7 @@ public class WrapUtil {
     private static void logDeprecation(int majorVersion) {
         DeprecationLogger.deprecateType(WrapUtil.class)
             .willBeRemovedInGradle9()
-            .withUpgradeGuideSection(7, "org_gradle_util_reports_deprecations")
+            .withUpgradeGuideSection(majorVersion, "org_gradle_util_reports_deprecations")
```

Comment:
:x: See the test comment. You're breaking the deprecation URL.

Reply:
actually not, If you go to https://docs.gradle.org/current/userguide/upgrading_version_8.html#org_gradle_util_reports_deprecations it is correct
whereas https://docs.gradle.org/current/userguide/upgrading_version_7.html#org_gradle_util_reports_deprecations is not there.

Reply:
The proper anchor is `org_gradle_util_reports_deprecations_7`, but it is there. There is no `WrapUtil` deprecation for 8.